src/components/@shared/Onboarding/HideButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactElement } from 'react'
+import Button from '../atoms/Button'
+import styles from './HideButton.module.css'
+import Tooltip from '../atoms/Tooltip'
+import Markdown from '../Markdown'
+import Cross from '@images/x-cross.svg'
+
+export default function HideButton({
+  setShowOnboardingModule,
```

Comment:
suggestion(blocking): see comment below: rather import this in the component via the hook